products-tools/index.md

```diff
@@ -1,6 +1,186 @@
+      </div>
+      
+      <div class="col-md-8">
+        <h3>
+          Nigeria MDGs Information System
+        </h3> NMIS visualizes data for schools, health clinics, and water pionts to facilitate data-driven planning across hundreds of Local Government Areas in Nigeria. Local planning and centralized review of local plans are facilitated by data about geo-located social infrastructure in a common map-based platform, and geographical visualizations and aggregations by geopolitical entities. 
+        
+        <br /> <a href="http://nmis.mdgs.gov.ng/" target="blank"> nmis.mdgs.gov.ng </a>
+      </div>
+    </div>
+  </div>
+</div>
+
+<div class="row-fluid products">
+  <div class="col-md-12">
+    <div class="row-fluid">
+      <div class="col-md-4">
+        <a href="http://revisit.global/"><img src="http://sel.columbia.edu/wp-content/uploads/2013/05/Revisit_Identity_Final.png" alt="Revisit_Identity_Final" /></a>
+      </div>
+      
+      <div class="col-md-8">
+        <h3>
+          Revisit
+        </h3> Data collection is essential to the development planning process and is a foundation of SEL’s work. Smartphones have made the collection of physical points quick and easy. However, in large, national-scale mapping efforts, it is very difficult to ensure complete coverage in a single collection effort and during multiple efforts to avoid duplication of points. Revisit, currently in development with funding from the Bill and Melinda Gates Foundation, aims to streamline the collection process as well as build a central facility repository that enables countries and organizations to share basic facility data—helping to coordinate aid and maintain a more complete and up-to-date picture of social infrastructure. 
+        
+        <br /> <a href="http://revisit.global/" target="blank"> revisit.global </a>
+      </div>
+    </div>
+  </div>
+</div>
+
+<div class="row-fluid products">
+  <div class="col-md-12">
+    <div class="row-fluid">
+      <div class="col-md-4">
+        <a href="http://spatialplanner.modilabs.org/"><img class="alignnone size-full wp-image-2589" alt="spatialplanner" src="http://modi.mech.columbia.edu/wp-content/uploads/2013/08/spatialplanner.png" width="250" height="112" /></a><img alt="" />
+      </div>
+      
+      <div class="col-md-8">
+        <h3>
+          Spatial Planner
+        </h3> Traditional planning techniques are population-based (eg. X health clinics for Y people). Spatial Planner makes resource planning geography aware: given geographical population distribution and locations of existing infrastructure, this web-based tool recommends where to put new facilities so that the most people are closest to social infrastructure. 
+        
+        <a href="http://spatialplanner.modilabs.org/" target="blank"> spatialplanner.modilabs.org </a> / <a href="http://github.com/modilabs/NeXT" target="blank"> source </a>
+      </div>
+    </div>
+  </div>
+</div>
```

Comment:
Oh I forgot to remove the hard coded ones


On Feb 22, 2016, 10:19 AM -0500, Viktor Roytman<notifications@github.com>, wrote:
>  
> Inproducts-tools/index.md(https://github.com/SEL-Columbia/SEL-Columbia.github.io/pull/67#discussion_r53638637):
>  
> >+<div class="col-md-12">>+<div class="row-fluid">>+<div class="col-md-4">>+<a href="http://spatialplanner.modilabs.org/"><img class="alignnone size-full wp-image-2589" alt="spatialplanner" src="http://modi.mech.columbia.edu/wp-content/uploads/2013/08/spatialplanner.png" width="250" height="112" /></a><img alt="" />>+</div>>+>+<div class="col-md-8">>+<h3>>+     Spatial Planner>+</h3>Traditional planning techniques are population-based (eg. X health clinics for Y people). Spatial Planner makes resource planning geography aware: given geographical population distribution and locations of existing infrastructure, this web-based tool recommends where to put new facilities so that the most people are closest to social infrastructure.>+>+<a href="http://spatialplanner.modilabs.org/" target="blank">spatialplanner.modilabs.org</a>/<a href="http://github.com/modilabs/NeXT" target="blank">source</a>>+</div>>+</div>>+</div>>+</div>
>  
> @Nasafato(https://github.com/Nasafato)what's with the hard-coded posts?
>  
>  
> —
> Reply to this email directly orview it on GitHub(https://github.com/SEL-Columbia/SEL-Columbia.github.io/pull/67/files#r53638637).
>  
>  
>